pkg/dynamiclistener/server.go

```diff
@@ -404,6 +424,21 @@ func (s *Server) getCertificate(hello *tls.ClientHelloInfo) (*tls.Certificate, e
 	return tlsCert, nil
 }
 
+func (s *Server) getRotateSettingInDays() (int64, error) {
+	setting, err := s.settingsLister.Get("", settings.RotateCertsIfExpiringInDays.Name)
```

Comment:
can we use `settings.RotateCertsIfExpiringInDays.Get()` directly?

Reply:
Oh I was under the impression that is getting settings from etcd, but it is using cache, will switch to that